packages/docs/src/components/nav-link.js

```diff
@@ -1,33 +1,26 @@
 /** @jsx jsx */
-import { jsx } from 'theme-ui'
-import { Link } from 'gatsby'
-import isAbsoluteURL from 'is-absolute-url'
+import { jsx } from "theme-ui"
+import { Link } from "gatsby"
+import isAbsoluteURL from "is-absolute-url"
 
 const styles = {
-  display: 'inline-block',
+  display: "inline-block",
   px: 2,
   py: 2,
-  color: 'inherit',
-  textDecoration: 'none',
+  color: "inherit",
+  textDecoration: "none",
   fontSize: 1,
-  fontWeight: 'bold',
-  '&.active': {
-    color: 'primary',
-  },
+  fontWeight: "bold",
+  "&.active": {
+    color: "primary"
+  }
 }
 
 export default ({ href, ...props }) => {
-  const isExternal = isAbsoluteURL(href || '')
+  const isExternal = isAbsoluteURL(href || "")
   if (isExternal) {
     return <a {...props} href={href} css={styles} />
   }
   const to = props.to || href
-  return (
-    <Link
-      {...props}
-      to={to}
-      css={styles}
-      activeClassName='active'
-    />
-  )
+  return <Link {...props} to={to} css={styles} activeClassName="active" />
```

Comment:
Is there a prettier option to always break props to separate lines yet?

Reply:
I don't think so :(. If there is, we add it to the config and apply to all files